### PR TITLE
feat(DQL): Rename gql package to dql

### PR DESCRIPTION
**Pull request by MichelDiz**
## Problem

GraphQL Spec uses the acronym GQL to reference its language in its packages. Dgraph used this acronym in the past because the language was called "GraphQL+-". But today as the name has been updated and we have native support for GraphQL. This becomes a problem as it confuses package names.

## Solution

Rename GQL package to DQL

### CODE REVIEW

**Review by matthewmcneely — APPROVED** [review]
LGTM... A welcomed change!

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,10 @@ gql/gql-fuzz.zip
 gql/fuzz-data/crashers
 gql/fuzz-data/suppressions
```
Comment (joshua-goldstein) [inline .gitignore:10]: here too?

**Comment by matthewmcneely** [comment]
This will likely *demolish* many in-process PR's. I'd recommend this gets merged into main immediately after we cut the next release (when more PRs will be settled)

**Comment by skrdgraph** [comment]
> This will likely _demolish_ many in-process PR's. I'd recommend this gets merged into main immediately after we cut the next release (when more PRs will be settled)

+1 ^ re-iterating Matthew's message

**Pull request merged**